db/migrate/20200512161742_remove_vmdb_database_settings.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveVmdbDatabaseSettings < ActiveRecord::Migration[5.2]
+  class SettingsChange < ActiveRecord::Base
+  end
+
+  def up
+    say_with_time('Removing RemoveVmdbDatabaseSettings') do
+      SettingsChange.where(:key => '/database/metrics_collection/collection_schedule').destroy_all
+      SettingsChange.where(:key => '/database/metrics_collection/daily_rollup_schedule').destroy_all
+      SettingsChange.where(:key => '/database/metrics_history/keep_daily_metrics').destroy_all
+      SettingsChange.where(:key => '/database/metrics_history/keep_hourly_metrics').destroy_all
+      SettingsChange.where(:key => '/database/metrics_history/purge_schedule').destroy_all
+      SettingsChange.where(:key => '/database/metrics_history/purge_window_size').destroy_all
+      SettingsChange.where(:key => '/database/metrics_history/queue_timeout').destroy_all
+      SettingsChange.where(:key => '/ui/display_ops_database').destroy_all
+      SettingsChange.where(:key => '/workers/worker_base/schedule_worker/log_database_statistics_interval').destroy_all
```

Comment:
As @Fryguy mentioned in the UI PR, there were some settings that could be in the database, so I added a removal of all of the settings removed in the backend PR.  See above.